sdk/ai/azure-ai-inference/azure/ai/inference/models/_patch.py

```diff
@@ -470,6 +471,33 @@ async def aclose(self) -> None:
         await self._response.close()
 
 
+class InputAudio(InputAudioGenerated):
+
+    @classmethod
+    def load(
+        cls,
+        *,
+        audio_file: str,
+        audio_format: str,
+    ) -> Self:
+        """
+        Create an InputAudio object from a local audio file. The method reads the audio
+        file and encodes it as a base64 string, which together with the audio format
+        is then used to create the InputAudio object passed to the request payload.
+
+        :ivar audio_file: The name of the local audio file to load. Required.
```

Comment:
I think quality gates will ask you to use `:keyword` here instead of `:ivar`. I forget which tool calls this out (mypy, pyright or pylint)